services/applicationService.js

```diff
@@ -0,0 +1,43 @@
+const { ProfileEducationModel } = require('../models/ProfileEducation');
```

Comment:
Move to helpers/ rather than services/
Services should be considered 3rd party.

Reply:
Oki doki (+ renaming)👍
![image](https://github.com/user-attachments/assets/1a957847-009d-46b4-b610-c9ffc8058905)